src/lib/dev-environment/dev-environment-core.js

```diff
@@ -59,14 +64,22 @@ type SQLImportPaths = {
 	inContainerPath: string
 }
 
-export async function startEnvironment( slug: string, options: StartEnvironmentOptions ) {
+type WordPressTag = {
+	ref: string;
+	tag: string;
+	cacheable: boolean;
+	locked: boolean;
+	prerelease: boolean;
+}
+
+export async function startEnvironment( slug: string, options: StartEnvironmentOptions ): Promise<void> {
 	debug( 'Will start an environment', slug );
 
 	const instancePath = getEnvironmentPath( slug );
 
 	debug( 'Instance path for', slug, 'is:', instancePath );
 
-	const environmentExists = fs.existsSync( instancePath );
+	const environmentExists = await fs.existsSync( instancePath );
```

Comment:
What is this change for? I would assume sync variant of he comman doesn't need await?

Reply:
Yeah, I was planning to use `fs.promises`, but `exists`/`existsSync` are deprecated and are not a part of the Promises API.